### PR TITLE
feat: Add Command bar for improved productivity

### LINKED ISSUE #3335: Command bar for productivity
The command bar is supported by most of the applications right now. The palette that opens when clicking on `Cmd + K` is really useful to navigate through the application. I've been using Superhuman for quite some time, I use the command bar more often than the keyboard shortcuts. 

Todos: 

- [ ] Research about the existing projects which can be integrated directly to Chatwoot (open-source as well as closed source ones like commandbar.com)
- [ ] List out the features that need to be supported. List all the global features as well route-aware commands.

### PR DESCRIPTION
**Pull request by pranavrajs**
Fixes #3335 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/javascript/dashboard/i18n/locale/en/generalSettings.json [hunk 1]
```diff
@@ -79,5 +79,39 @@
...
     }
+  },
+  "COMMAND_BAR": {
+    "SECTIONS": {
+      "GENERAL": "General",
+      "REPORTS": "Reports",
+      "CONVERSATION": "Conversation",
+      "CHANGE_ASSIGNEE": "Change Assignee",
+      "CHANGE_TEAM": "Change Team",
+      "ADD_LABEL": "Add label to the conversation",
```
Comment (nithindavid) [inline app/javascript/dashboard/i18n/locale/en/generalSettings.json:90]:
```suggestion
      "ADD_LABEL": "Add labels to the conversation",
```

File: app/javascript/dashboard/routes/dashboard/commands/commandbar.vue [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+<script>
+import 'ninja-keys';
+import conversationHotKeysMixin from './conversationHotKeys';
+import goToCommandHotKeys from './goToCommandHotKeys';
+
+export default {
+  mixins: [conversationHotKeysMixin, goToCommandHotKeys],
+  data() {
+    return {
+      placeholder: 'Search or jump to',
```
Comment (nithindavid) [inline app/javascript/dashboard/routes/dashboard/commands/commandbar.vue:20]: i18n ?

**Review by nithindavid — CHANGES_REQUESTED** [review]
Has a minor comment, otherwise looks good.

File: app/javascript/dashboard/routes/dashboard/commands/commandbar.vue (hunk 2 above)
Comment (pranavrajs) [reply app/javascript/dashboard/routes/dashboard/commands/commandbar.vue:20]: Yes, we need to, will move that.

**Review by nithindavid — APPROVED** [review]

**Comment by nithindavid** [comment]
@pranavrajs Can you fix the code climate issues ?

**Review by nithindavid — APPROVED** [review]

**Review by muhsin-k — APPROVED** [review]

**Pull request merged**